stream-chat-android-offline/src/main/java/io/getstream/chat/android/livedata/ChatDomain.kt

```diff
@@ -91,7 +91,7 @@ public sealed interface ChatDomain {
     public val banned: LiveData<Boolean>
 
     /** The retry policy for retrying failed requests */
-    public var retryPolicy: RetryPolicy
```

Comment:
Is there a good reason why we're not just deprecating the setter first, and introducing a breaking change instead?

Reply:
We don't provide any documentation for that. We don't even use it everywhere for retries inside. It's weird also to reset it after ChatDomain is configured already. I have doubts that it's used by anyone currently.

Reply:
Given that setting that property was previously the only way to set this, I'd be happier if we just deprecated it at first as we introduced the new API that replaces it, but if you feel like this'd be a super niche feature, I'm okay with doing the breaking change too, given that it's clearly highlighted in the changelog